dd-client/io_runner.sh

```diff
@@ -1,27 +1,31 @@
 #!/bin/bash
 
-#######################################################################################################################
-# Script Name   : io_runner.sh         									      		
-# Description   : Run dd profiles on the /datadir. 
-# Creation Data : 12/10/2018                                                                                          
-# Modifications : None											               		
-# Script Author : Sudarshan					      
-#######################################################################################################################
-
-TEST_SIZE="512k"
-TEST_COUNT="1000"
+blockSize=$(echo $BLOCK_SIZE)
```

Comment:
several of these  ENV (app , mount info) seem to be mandatory requirements for the io_runner script. can we have a check to see that they are not empty & if so, error out? (guess these are provided by default by the calling litmusbook -- however, for the sake of completeness - considering this may run as standalone job too -- can we have this check)